src/server/users/controllers.js

```diff
@@ -2,7 +2,7 @@ import { KeyUtils, ENTITY, STATE } from '../key-utils';
 import { S3Connector } from '../aws-s3';
 import db from '../database';
 import { APIError } from '../ErrorHandler';
-import { CreatedPostManager, SavedPostManager } from './models';
+import { SavedPostManager, CreatedPostManager } from './models';
```

Comment:
When I removed 'addCreatedPost' function in controller, I needed to remove CreatedPostManager import.
After then I added CreatedPostManager import again to implement getCreatedPost function.
Not that big reason lol